Source/VCGeneration/Split.cs

```diff
@@ -1340,6 +1340,13 @@ public void ReadOutcome(ref ConditionGeneration.Outcome curOutcome, out bool pro
               curOutcome = ConditionGeneration.Outcome.Inconclusive;
             }
 
+            return;
+          case ProverInterface.Outcome.Canceled:
+            if (curOutcome != ConditionGeneration.Outcome.Canceled)
+            {
+              curOutcome = ConditionGeneration.Outcome.Canceled;
+            }
```

Comment:
Why not just set it unconditionally?  Is it a setter?

Reply:
Good catch. I tried to mimick what I saw before, but apparently I did not understand what they were doing. Now I do: Let's assign it unconditionally.